services/f-droid/f-droid.service.js

```diff
@@ -1,31 +1,43 @@
 'use strict'
 
+const Joi = require('joi')
 const BaseService = require('../base')
+const validate = require('../../lib/validate')
 const { addv: versionText } = require('../../lib/text-formatters')
 const { version: versionColor } = require('../../lib/color-formatters')
-const { InvalidResponse } = require('../errors')
+const { InvalidResponse, InvalidParameter } = require('../errors')
 
 module.exports = class FDroid extends BaseService {
-  async fetch({ appId }) {
+  async fetch({ appId, format }) {
     // currently, we only use the txt format. There are few apps using the yml format.
```

Comment:
Could you update this comment to describe the new behavior? It should make the rest of this easier to follow for an unfamiliar reader.